.github/issue-branch.yml

```diff
@@ -0,0 +1,17 @@
+silent: true
+branchName: '${issue.title,}'
```

Comment:
realmente precisa da virgula? :thinking:

Reply:
https://www.linkedin.com/feed/update/urn:li:activity:7026918452665757696/

The comma is necessary by lowercased our feature branchs.